database/postgres/csv.js

```diff
@@ -0,0 +1,37 @@
+
+const fs = require('fs');
+const utils = require('./utils');
+const fastcsv = require('fast-csv');
+
+const ws_product = fs.createWriteStream("products.csv");
+const ws_reviews = fs.createWriteStream("reviews.csv");
+const ws_users  = fs.createWriteStream("users.csv");
+const ws_features  = fs.createWriteStream("features.csv");
+const ws_productFeatures = fs.createWriteStream("productFeatures.csv");
+
+let products = utils.createProducts()
+let reviews = utils.createReviews()
+let users = utils.createUsers()
+let features = utils.createFeatures()
+let productFeatures = utils.createProductFeatures()
+
```

Comment:
Looks great! One thing to watch out for is that when doing 10 million records you'll have to append to your csv files. I'd checkout out how to append data to csv files and which options you can put in the options object, which currently reads {headers: true} and see how you can append data so you can load your data to the files in chunks